### PR TITLE
Feature/co 1852 implement get /services

### PR DESCRIPTION
**Pull request by kalcicd**

### CODE REVIEW

File: src/db/awsES/services-dao.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import _ from 'lodash';
+import AWS from 'aws-sdk';
```
Comment (Ruefa) [inline src/db/awsES/services-dao.js]: Fix capitalization like in the other PR

File: src/db/awsES/services-dao.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import config from 'config';
+import connectionClass from 'http-aws-es';
+import esb from 'elastic-builder';
+import elasticsearch from 'elasticsearch';
```
Comment (Ruefa) [inline src/db/awsES/services-dao.js]: Alphabetize input order

File: src/db/awsES/services-dao.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  const client = elasticsearch.Client({
+    host: domain,
+    log: 'error',
+    connectionClass,
+    awsConfig: new AWS.Config({
+      accessKeyId,
+      secretAccessKey,
+      region: 'us-east-2',
+    }),
+  });
```
Comment (Ruefa) [inline src/db/awsES/services-dao.js]: I think this code should be a function in `connection` that we call to get a client. Like how we do with oracleDB

File: src/db/awsES/services-dao.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  const res = await client.search({
+    index: 'services',
+    body: buildQueryBody(queryParams),
+  });
+
+  const services = [];
+  // eslint-disable-next-line dot-notation
+  _.forEach(res.hits.hits, (service) => services.push(service['_source']));
```
Comment (Ruefa) [inline src/db/awsES/services-dao.js]: See my comment https://github.com/osu-mist/locations-api/pull/71#discussion_r461940409

File: src/serializers/services-serializer.js [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    topLevelSelfLink,
+    query,
+    enableDataLinks: true,
+  };
+
+  // flatten attributes object in rawServices for serializer
+  const newRawServices = [];
+  _.forEach(rawServices, (rawService) => {
+    newRawServices.push({
+      ...{ id: rawService.id, type: rawService.type },
```
Comment (Ruefa) [inline src/serializers/services-serializer.js]: I could be wrong here but I doesn't `...{}` cancel each other out here? I think you can do 
```
    newRawServices.push({
      id: rawService.id,
      type: rawService.type,
```

**Review by Ruefa — CHANGES_REQUESTED** [review]

File: src/serializers/services-serializer.js (hunk 5 above)
Comment (kalcicd) [reply src/serializers/services-serializer.js]: you're right!

**Review by tsoliangwu0130 — APPROVED** [review]

**Pull request merged**